### PR TITLE
@benisgold/profiles analytics

### PR DESCRIPTION
**Pull request by benisgold**
Fixes RNBW-2856

## What changed (plus any additional context for devs)
- Added event analytics for ens transactions along with user properties for profiles

## Final checklist

- [x] Assigned individual reviewers?
- [x] Added labels?

### CODE REVIEW

File: src/components/header/DiscoverHeaderButton.js [hunk 1]
```diff
@@ -50,10 +51,14 @@ export default function DiscoverHeaderButton() {
     navigate,
   ]);
 
-  const onLongPress = useCallback(
-    () => navigate(Routes.PROFILE_SHEET, { address: 'mikedemarais.eth' }),
-    [navigate]
-  );
+  const onLongPress = useCallback(() => {
+    navigate(Routes.PROFILE_SHEET, { address: 'mikedemarais.eth' });
+    analytics.track('Viewed ENS profile', {
```
Comment (estebanmino) [inline src/components/header/DiscoverHeaderButton.js:56]: seems like we only use this method in __DEV__ so we shouldn't need it
image

File: src/screens/WalletScreen.js [hunk 2]
```diff
@@ -132,6 +139,12 @@ export default function WalletScreen() {
     userAccounts,
   ]);
 
+  useEffect(() => {
+    if (profilesEnabled) {
+      trackENSProfile();
```
Comment (estebanmino) [inline src/screens/WalletScreen.js:144]: 💯

**Review by estebanmino — APPROVED** [review]
💯 💯 💯 💯 💯

**Review by jxom — APPROVED** [review]
looks great

**Pull request merged**